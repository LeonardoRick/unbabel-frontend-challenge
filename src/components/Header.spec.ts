import HeaderVue from '@/components/Header.vue';
import { mount } from '@vue/test-utils';

describe('Header', () => {
    it('should mount', () => {
        expect(mount(HeaderVue).html()).toMatchSnapshot();
    });
});
