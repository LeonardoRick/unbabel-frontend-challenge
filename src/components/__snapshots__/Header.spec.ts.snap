// Vitest Snapshot v1

exports[`Header > should mount 1`] = `
"<header>
  <div class=\\"inner-container\\">
    <h1>
      <routerlink to=\\"/home\\">Transcriptions</routerlink>
    </h1>
    <nav>
      <routerlink class=\\"about\\" to=\\"/another-page\\">About</routerlink><button type=\\"button\\" class=\\"reset-button-style\\"><img src=\\"/src/assets/images/upload.svg\\" alt=\\"upload list of transcriptions\\"></button><button type=\\"button\\" class=\\"reset-button-style\\"><img src=\\"/src/assets/images/fetch-document.svg\\" alt=\\"fetch transcriptions\\"></button>
    </nav>
  </div>
</header>"
`;
